.travis.yml

```diff
@@ -14,16 +14,26 @@ dist: trusty
 compiler: clang # per: https://docs.travis-ci.com/user/languages/c
 # https://docs.travis-ci.com/user/environment-variables
 env:
- - CXX=clang++ DEBUG='nightmare*'
- # - CXX=clang++ DISPLAY=':99.0' DEBUG='nightmare*'
-# according to travis docs, the services -xvfb line will set DISPLAY
 branches:
   only:
     - master # https://github.com/okTurtles/group-income-simple/issues/58
 addons:
   apt:
     packages:
       - xvfb
+cache:
+  # Caches $HOME/.npm when npm ci is default script command
+  directories:
+    # cache folder with Cypress binary to avoid downloading Cypress again and again
+    - ~/.cache
```

Comment:
Yes you should follow the instructions here to be certain that `node_modules` and `~/.npm` are cached:

- https://docs.travis-ci.com/user/caching/#configuration
- https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#caching-with-npm